assets/css/src/_blocks.css

```diff
@@ -109,87 +109,12 @@ ul.wp-block-latest-posts.is-grid.alignfull {
 # Gallery Block - overrides core styles
 --------------------------------------------------------------*/
 
-.wp-block-gallery {
-	display: grid;
-	grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
-	grid-gap: 0.5em;
-}
-
-.wp-block-gallery:not(.components-placeholder) {
-	margin: 1.5em auto;
-}
-
-/* Override standard ul indentation. */
-
-.entry-content .wp-block-gallery {
-	padding-right: 1.5em;
-	padding-left: 1.5em;
-}
-
-@media screen and (min-width: 48em) {
-
-	.entry-content .wp-block-gallery {
-		padding-right: 0;
-		padding-left: 0;
-	}
+figure.wp-block-gallery {
+	margin-right: auto;
+	margin-left: auto;
 }
 
-.wp-block-gallery.columns-2 .blocks-gallery-item,
-.wp-block-gallery.columns-3 .blocks-gallery-item,
-.wp-block-gallery.columns-4 .blocks-gallery-item,
-.wp-block-gallery.columns-5 .blocks-gallery-item,
-.wp-block-gallery.columns-6 .blocks-gallery-item,
-.wp-block-gallery.columns-7 .blocks-gallery-item,
-.wp-block-gallery.columns-8 .blocks-gallery-item,
-.wp-block-gallery.columns-9 .blocks-gallery-item {
-	grid-column: span 1;
+.entry-content .blocks-gallery-grid {
```

Comment:
It think it would be better to use `.wp-block-gallery .blocks-gallery-grid` instead of `.entry-content .blocks-gallery-grid` . The margin will be applied in other contexts than the post content.

Reply:
We should also keep some alignment adjustments (padding) for backward compatibility. Existing posts will contain the old gallery block markup until they are updated.